File: Topcoder/SRM 232 DIV 2/src/BritishCoins.java

public class BritishCoins {

  public int[] coins(int pence) {
    int pounds = pence / (12 * 20);
    pence -= pounds * (12 * 20);
    int shillings = pence / 12;
    pence -= shillings * 12;
    return new int[]{pounds, shillings, pence};
  }
}

File: online_judges/Topcoder/Python/BritishCoins.py
class BritishCoins:
  def coins(self, pence):
    conv = (12 * 20, 12, 1)
    result = [0, 0, 0]
    for i in xrange(len(conv)):
      result[i] = (pence / conv[i])
      pence -= conv[i] * result[i]
    return tuple(result)

print BritishCoins().coins(533)